app/uk/gov/hmrc/gform/formstatistics/FormStatisticsModule.scala

```diff
@@ -40,10 +38,7 @@ class FormStatisticsModule(
     new Repo[Form](
       "forms",
       mongoModule.mongoComponent,
-      _._id.value,
-      Seq(
-        IndexModel(ascending("data.form.formTemplateId"), IndexOptions().name("formTemplateIdIdx").background(true))
-      )
```

Comment:
Why is this being deleted?

Reply:
I moved it to [here](https://github.com/hmrc/gform/pull/1083/files#diff-7f96266b152d406dc1e8b25d4e3627ec5652a96d0be2322e728614878fa5d2a5R226) to create all indexes the same place.